examples/tests/valid/std_containers.w

```diff
@@ -14,17 +14,15 @@ mut_map.set("five", 5);
 let immut_map = mut_map.copy();
 
 let s: str = s_array.at(1);
-let handler = inflight (body: str): str => {
-  assert(s == "two");
-  assert(s_array.at(1) == "two");
-  assert(s_array.length == 2);
-  assert(immut_array.length == 3);
+assert(s == "two");
+assert(s_array.at(1) == "two");
+assert(s_array.length == 2);
+assert(immut_array.length == 3);
 
-  assert(s_set.has("one"));
-  assert(s_set.size == 2);
-  assert(immut_set.size == 3);
+assert(s_set.has("one"));
+assert(s_set.size == 2);
```

Comment:
Didn't we want to normalize to `len`?

```suggestion
assert(s_set.len == 2);
```

Reply:
Yes, we will do it soon, I promise :)
Probably within a few days, but not in this PR - here I was just simplifying one of the tests because it had some extra boilerplate.